migration/src/spreadsheets.rs

```diff
@@ -171,13 +176,19 @@ impl SheetResult {
                     doc_string = row_values.next();
                 }
 
+                let mut intro_or_genre = Body;
+                if is_intro == true {
+                    intro_or_genre = Intro;
+                }
+
                 let new_chapter = Chapter {
                     index_in_parent: index_i64,
                     url_slug: chapter_url_slug,
                     chapter_name: cur_chapter_name,
                     document_short_name: doc_string,
                     id: None,
                     wordpress_id: wp_id,
+                    chapter_type: intro_or_genre,
```

Comment:
Mind matching the new field to the database column name?

Reply:
fixed!